lib/axlsx.rb

```diff
@@ -35,9 +35,9 @@
 
 begin
   if Gem.loaded_specs.has_key?("axlsx_styler")
-    raise StandardError.new("Please remove `axlsx_styler` from your Gemfile, the associated functionality is now built-in to `caxlsx` directly.")
+    raise StandardError, "Please remove `axlsx_styler` from your Gemfile, the associated functionality is now built-in to `caxlsx` directly."
   end
-rescue
+rescue StandardError => _
```

Comment:
`=> _` should be redundant

```suggestion
rescue StandardError
```

Reply:
Fixed